.travis.yml

```diff
@@ -1,52 +1,44 @@
 language: python
-python: '2.7'
+python: '3.6'
 
 branches:
   only:
   - master
   - develop
-  - feature/add_travis_deploy
+  - feature/travis_deploy_for_1.0
 
 virtualenv:
   system_site_packages: true
 
 matrix:
   include:
     - os: linux
-      dist: trusty
+      dist: bionic
       addons:
         apt:
           packages:
-          - pstoedit
-          - pdf2svg
-          - inkscape
-          - python-gtk2-dev
-          - imagemagick
-          - texlive-latex-base
-
-    - os: linux
-      dist: xenial
-      addons:
-        apt:
-          packages:
-          - pstoedit
-          - pdf2svg
-          - inkscape
-          - python-gtk2-dev
-          - imagemagick
+          # Will still load inkscape 0.92.x, hence use manual app image, see below
+          #- inkscape
+          - python3-gi
+          - python3-gi-cairo
+          - gir1.2-gtk-3.0
           - texlive-latex-base
       env:
         - RELEASE_MAKER=true
 
 before_script:
-   - python2 -m pip install pytest lxml Pillow
-   - python2 test_installation_script.py 2> /dev/null
-   - python2 setup.py
+   # Deactivated until tests are rewritten
+   #- python -m pip install pytest lxml Pillow
+   - wget -c https://inkscape.org/de/gallery/item/14204/Inkscape-883c7bc-x86_64.AppImage
+   - chmod +x Inkscape-883c7bc-x86_64.AppImage
+   - sudo mv Inkscape-883c7bc-x86_64.AppImage /usr/local/bin
```

Comment:
1) `sudo` is not required here. Executable can be placed in `$HOME/bin`. Maybe `$PATH` should be updated as well.

2) I would rename `Inkscape-883c7bc-x86_64.AppImage` to `inkscape.alpha`

Reply:
Thanks! Indeed $PATH needs to be updated so requirements.py finds the inkscape.alpha executable.